packages/point-intercept/jest.config.js

```diff
@@ -0,0 +1,5 @@
+module.exports = {
```

Comment:
all  test related stuff can go - it's all configured at the mono-repo root.

Reply:
Taken care of!